services/authenticator.js

```diff
@@ -147,13 +148,11 @@ function Authenticator() {
     try {
       await api.createUser(authConfig);
     } catch (error) {
-      if (error.message === 'Conflict') {
-        logger.error(`Your account already exists. Please, use the command ${chalk.cyan('lumber run lumber-forestadmin:login')}.`);
-      } else {
-        logger.error(`An unexpected error occured. Please create a Github issue with following error: ${chalk.red(error)}`);
-      }
+      const message = error.message === 'Conflict'
+        ? `Your account already exists. Please, use the command ${chalk.cyan('lumber run lumber-forestadmin:login')}.`
```

Comment:
**This** account already exists. Please, use the command ${chalk.cyan('lumber run lumber-forestadmin:login')} **to login with this account**.